app/web_conn.go

```diff
@@ -201,20 +208,25 @@ func (wc *WebConn) writePump() {
 				continue
 			}
 
-			var msgBytes []byte
+			buf.Reset()
+			var err error
 			if evtOk {
 				cpyEvt := evt.SetSequence(wc.Sequence)
-				msgBytes = []byte(cpyEvt.ToJson())
+				err = cpyEvt.Encode(enc)
 				wc.Sequence++
```

Comment:
Unrelated to the PR and probably a minor thing but should the sequence get bumped also in case of error?

Reply:
Hmm.. good point. I think it should not. But I need to verify a bit how the sequence number is used by the client. In any case, let's visit this in a separate PR.